modules/tensor_mechanics/test/tests/tangent_operator/tests

```diff
@@ -0,0 +1,39 @@
+[Tests]
```

Comment:
Just a couple of small things:
1) Could you re-name this directory to something like `creep_tangent_operator` just so it's more readily apparent what model's tangent operator you are testing?
2) There is probably a lot of duplication in these test input files. We've been using `cli_args` a lot to make a few targeted changes to an input file to re-use the same input file to test models in different ways. You can add or overwrite parameters using that, but you can't remove parameters that were set in the original file. Could you look into doing that on this test if you think that makes sense?  I'm happy to help you with that if you have questions.

Reply:
Good recommendations. I tend to like `cli_args`, but I thought that most didn't. It's now updated following your recommendations.